### PR TITLE
Add Google Earth Engine icon

### LINKED ISSUE #8912: add Google Earth Engine
### Brand Name

earthengine

### Website

https://earthengine.google.com/

### Popularity Metric

the Earthengine website have become the GoTo tool for remote sensing analysis in the past 10 years, progressively overcoming all its competitors. No analytics can be provided on the website as only the commercial link can be reached: https://earthengine.google.com/

The API is available on Github but only as a mirror: https://github.com/google/earthengine-api but still reached 2500 stars. 

If you look at the associated publication it's the most cited article in the field of remote-sensing: https://scholar.google.com/citations?view_op=view_citation&hl=fr&user=jC_M0KYAAAAJ&citation_for_view=jC_M0KYAAAAJ:X4-KO54GjGYC

### Official Resources for Icon and Color

https://earthengine.google.com/static/images/earth-engine-logo.png

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by 12rambau**
**Issue:** closes #8912

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I picked the same hex value as per Google Earth as the logo is simply a cog around the Google Earth logo. The rest was fairly simple the file send by Noel was already optimized, I simply resized it for 24x24 canvas.

googleearthengine

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -4901,6 +4901,19 @@
...
+            "hex": "4285F4",
+            "source": "https://earthengine.google.com",
+            "aliases": {
+                "aka": [
+                    "Earth Engine",
+                    "earthengine",
+                    "GEE",
+                    "EE"
+                ]
+            }
```
Comment (adamrusted) [inline _data/simple-icons.json:4915]: I don't think we need all these aliases. You can probably just get away with the main title :)

File: _data/simple-icons.json [hunk 2]
```diff
@@ -4901,6 +4901,11 @@
             "hex": "4285F4",
             "source": "https://earth.google.com/web/"
         },
+        {
+            "title": "Google Earth Engine",
+            "hex": "4285F4",
+            "source": "https://earthengine.google.com"
```
Comment (adamrusted) [inline _data/simple-icons.json:4907]:
```suggestion
            "source": "https://github.com/simple-icons/simple-icons/issues/8912#issuecomment-1638850020"
```

**Review by adamrusted — COMMENTED** [review]
Given this was suggested by the GEE team, I'm happy to accept on that front. I'd just change the source to the specific issue comment where you mentioned them suggesting it - that way we know for future reference!

**Comment by dirien** [comment]
@adamrusted do you need a review?

**Comment by adamrusted** [comment]
Yes please @dirien 🚀

**Review by service-paradis — APPROVED** [review]
LGTM 🎉

**Pull request merged**